doc/world_builder_declarations_open.md

```diff
@@ -373,8 +373,8 @@
 - **type**:array
 - **additionalProperties**:false
 - **minItems**:1
-- **maxItems**:2
-- **description**:
+- **maxItems**:2.0
+- **documentation**:
```

Comment:
I think this means that you generated the output with an older version. Can you rebase to the current main and rerun the tests to generate a new version of this file?